yt_dlp/extractor/saitosan.py

```diff
@@ -0,0 +1,83 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class SaitosanIE(InfoExtractor):
+    IE_NAME = 'Saitosan'
```

Comment:
It is not necessary to define `IE_NAME` when it is same as the class name. Ofc, no harm in defining it either